src/video/win32_v.cpp

```diff
@@ -670,6 +674,24 @@ LRESULT CALLBACK WndProcGdi(HWND hwnd, UINT msg, WPARAM wParam, LPARAM lParam)
 			return TRUE;
 		}
 
+		case WM_DPICHANGED: {
+			auto did_adjust = AdjustGUIZoom();
+
+			/* Resize the window to match the new DPI setting. */
+			RECT* const prcNewWindow = (RECT*)lParam;
```

Comment:
Not sure actually want a const pointer to a modifiable RECT, and it is not reused anyway, so just (and also formatting):
```suggestion
			RECT *prcNewWindow = (RECT *)lParam;
```

Reply:
I took the sample code from here: https://learn.microsoft.com/en-us/windows/win32/hidpi/wm-dpichanged, but I'll change it regardless.